workflows/update-deps.yml

```diff
@@ -57,20 +57,39 @@ jobs:
           fi
           echo "BRANCH_TARGET=$BRANCH_TARGET" >> $GITHUB_ENV
 
+          SHOULD_BUMP=$(echo '${{ env.CONFIG }}' | jq -r ".bump")
+          if [[ "$SHOULD_BUMP" == "true" ]]; then
+            echo "DEPS_BUMP_METHOD=npm-check" >> $GITHUB_ENV
+          else
+            echo "DEPS_BUMP_METHOD=npm_update" >> $GITHUB_ENV
+          fi
```

Comment:
I'm wondering, can't we just merge both workflows, and depending on the config, they will update minor or major versions?

Reply:
As we discussed earlier, we will stick to separate versions due to schedule differences between those two workflows.